src/styles/index.scss

```diff
@@ -1,3 +1,77 @@
 body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vmin;
```

Comment:
Using `vmin` for height might not always be the best choice depending on the design requirements. If the design is supposed to be responsive and cover the full viewport height, consider using `100vh` instead of `100vmin` to ensure it covers the full height of the viewport on all devices.